docs/design/fabric-v2+/fpc-lifecycle-v2.puml

```diff
@@ -15,124 +15,149 @@ title Foot Box removed
+    Peer_CLI -> Peer1++ : install chaincode from package
+    return packageId
+
+    note right Peer_CLI
+    The following bookkeeping operations help remember
+    (across lifecycle calls) that a package (id)
+    corresponds to a FPC chaincode. Similarly, the
+    approve will later store data for an FPC chaincode
+    installed locally, and the commit checks for data
+    of a locally-approved FPC chaincode.
+
     Also, it should be noted that
     - the install lifecycle op is peer-specific, while
-    - the approveformyorg is org-specific, to be run on at most one org's peer
+    - the approveformyorg is org-specific, to be run
+    on at most one org's peer
     - the commit op has to be run on a single peer
-    The current bookkeeping ops expect the approve and commit ops to
-    run on peers where the FPC chaincode has been previously installed.
-end note
-FPC_Peer1_CLI_Wrapper -> FPC_Peer1_CLI_Wrapper : retrieve cc_language from package
-FPC_Peer1_CLI_Wrapper -> FPC_Peer1_CLI_Wrapper : if cc_language==fpc then store packageId
-return packageId /' Admin -> FPC_Peer1_CLI_Wrapper '/
-end /' group install '/
-
-ref over Admin, FPC_Peer2_CLI_Wrapper, Peer2_CLI
-    install chaincode on Peer2
-end /' ref '/
-
-group local approve
-Admin -> FPC_Peer1_CLI_Wrapper ++: lifecycle chaincode approveformyorg
-FPC_Peer1_CLI_Wrapper -> Peer1_CLI ++: lifecycle chaincode approveformyorg
-note right
-    **Chaincode Definition for FPC Chaincode**
+
+    The current bookkeeping ops expect the approve and
+    commit ops to run on peers where the FPC chaincode
+    has been previously installed.
+    end note
+
+    Peer_CLI -> Peer_CLI : retrieve cc_language from package
+    Peer_CLI -> Peer_CLI : if cc_language==fpc\n\tthen store packageId
+    return packageId
+end
+
+ref over Admin, Peer_CLI, Peer2
+    Admin installs chaincode on Peer2
+end
+
+group approveformyorg
+    Admin -> Peer_CLI++ : lifecycle\nchaincode approveformyorg
+    note right
+    **Chaincode Definition for FPC Chaincode** including
     Name: cc_name,
     Version: **mrenclave**,
-    Sequence: sequence_number, Endorsement Policy: (2of3),
-    Plugins: <ecc validation plugin>
-    NOTE: the endorsement policy specifies how many enclaves are expected to run the chaincode.
-    In the initial version of FPC, a specific FPC chaincode will be implemented to run on
-    a "designated" peer. Hence, initially, the endorsement policy will require a single endorser.
-    FPC is expected to support multiple endorsers in the future. To have an idea of how FPC will
-    enalve multiple endorsers, please check out the diagram describing
-    how secret/private keys are expected to be distributed among endorsing chaincode enclaves.
+    Sequence: sequence_number,
+    Endorsement Policy: (2of3),
+    Plugins: ecc-vscc (FPC validation plugin)
+
+    NOTE: the admin approves the chaincode definition
+    including the **mrenclave** that was computed during
+    code inspection.
+
```

Comment:
Not immediately clear what the point of this is. (Invocation of) approvefororg is by definition the approval of the chaincode, so that this also covers mrenclave should be clear from above. At least as-is i think the text is more confusing rather than further illuminating. So i would drop this.
```suggestion

```